src/NUnitFramework/nunitlite.tests/CommandLineTests.cs

```diff
@@ -172,7 +172,7 @@ public void CanRecognizeLowerCaseOptionValues(string propertyName, string option
 
             foreach (string canonicalValue in canonicalValues)
             {
-                string lowercaseValue = canonicalValue.ToLower(CultureInfo.InvariantCulture);
+                string lowercaseValue = canonicalValue.ToLower();
```

Comment:
Yes, probably, thanks. It was late and this was just a test, so I didn't spend much time on it 😄 

I will push the change.